### PR TITLE
BZ#1613747 - updated disabling a build strategy

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1613747

### CODE REVIEW

**Comment by wewang58** [comment]
according the doc pr, Tested in openshift v3.9.40 and openshift version: v3.11.0-0.14.0, it works

File: admin_guide/securing_builds.adoc [hunk 1]
```diff
@@ -66,13 +66,41 @@ Default roles are provided which grant the *create* permission on the build stra
 == Disabling a Build Strategy Globally
 To prevent access to a particular build strategy globally, log in as a user with
 xref:../architecture/additional_concepts/authorization.adoc#roles[*cluster-admin*]
-privileges and remove the corresponding role from the *system:authenticated* group:
+privileges, remove the corresponding role from the *system:authenticated* group, and apply the annotation `openshift.io/reconcile-protect: "true"` to protect them from changes between the API restarts. The following example shows disabling the docker build strategy.
 
+. Apply `openshift.io/reconcile-protect` annotation
```
Comment (kalexand-rh) [inline admin_guide/securing_builds.adoc]: s/`openshift.io/reconcile-protect` annotation/the `openshift.io/reconcile-protect` annotation:

File: admin_guide/securing_builds.adoc [hunk 2]
```diff
@@ -66,13 +66,41 @@ Default roles are provided which grant the *create* permission on the build stra
...
+  name: system:build-strategy-docker
+subjects:
+- kind: SystemGroup
+  name: system:authenticated
+userNames:
+- system:serviceaccount:management-infra:management-admin
+----
+<1> Change the `openshift.io/reconcile-protect` annotation's value to `"true"`. By default, it is set to `"false"`.
+
+. Remove the role
```
Comment (kalexand-rh) [inline admin_guide/securing_builds.adoc]: s/role/role:

**Review by kalexand-rh — COMMENTED** [review]
couple of picks, but LGTM!

**Pull request merged**